src/cljs/rems/fields.cljs

```diff
@@ -335,9 +337,15 @@
       {:info-id info-id
        :aria-label-text (text :t.form/upload-extensions)
        :focus-when-collapse-opened focus-when-collapse-opened
-       :body-text [:span [text :t.form/upload-extensions]
-                   ": "
-                   attachment-types/allowed-extensions-string]}]]))
+       :body-text [:<>
+                   [:div
+                    [:span [text :t.form/upload-extensions]
+                     ": "
+                     attachment-types/allowed-extensions-string]]
+                   [:div
+                    [:span [text :t.form/attachment-max-size]
+                     ": "
+                     (:attachment-max-size config)]]]}]]))
```

Comment:
reminder to self: add `format-file-size` here as well

Reply:
And this could use `text-format` too.